pkg/proxy/OWNERS

```diff
@@ -1,18 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- thockin
+- sig-network-approvers
 - matchstick
 reviewers:
-- thockin
+- sig-network-reviewers
 - lavalamp
```

Comment:
we should probably remove everyone here and offer @m1093782566 a spot in reviewers.

Reply:
Thanks Tim :)

Reply:
@thockin I added @m1093782566 to sig-network-reviewers; he was already in the list for pkg/proxy/OWNERS so I assume you meant to just add him to signet review.